### PR TITLE
fix eslint config

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

**Comment by baomastr** [comment]
I think we don't eslint:recommended because main rules are handling by airbnb config

File: src/containers/App.tsx [hunk 1]
```diff
@@ -17,11 +17,9 @@ import MarketDetail from 'containers/Main/MarketDetail';
 import VoteOverview from 'containers/Main/VoteOverview';
 import ProposerDetail from 'containers/Main/ProposerDetail';
 import VoterLeaderboard from 'containers/Main/VoterLeaderboard';
-import VrtConversion from 'containers/Main/VrtConversion';
 import Transaction from 'containers/Main/Transaction';
 import Theme from './Theme';
 
-import 'assets/styles/App.scss';
```
Comment (coreyar) [inline src/containers/App.tsx:24]: This scss file is being used so we need to keep it

Comment (baomastr) [reply src/containers/App.tsx:24]: but it's already imported in index.scss

Comment (coreyar) [reply src/containers/App.tsx:24]: I see that now, imported in `src/index.tsx`

**Pull request merged**